vuepress/docs/next/docs/curate/local-hub-bundle-versions-faq.md

```diff
@@ -29,11 +29,11 @@ spec:
 ```
 
 As you can see from the code above, the bundle `cms-quickstart-bundle` has 2 versions available.
-Each version in the `tags` object must correspond a tag in the git repository provided in the `tarball` field.
+Each version in the `tags` object must correspond a tag in the Git repository provided in the `tarball` field.
```

Comment:
TODO re v5?

Reply:
yes, TODO... the example above cites git links so we'll have to revisit this to edit thoroughly/correctly